bambi/families/family.py

```diff
@@ -28,13 +28,31 @@ class Family:
     Parameters
     ----------
     name : str
-        Family name.
+        The name of the family. It can be any string.
     likelihood: Likelihood
-        A ``Likelihood`` instace specifying the model likelihood function.
+        A ``bambi.families.Likelihood`` instace specifying the model likelihood function.
     link : str or Link
-        The name of the link function, or the function itself, transforming the linear model
-        prediction to the mean parameter of the likelihood. If a function, it must be able to
-        operate over theano tensors rather than numpy arrays.
+        The name of the link function or a ``bambi.families.Link`` instance. The link function
+        transforms the linear model prediction to the mean parameter of the likelihood funtion.
+
+    Examples
+    --------
+    Some imports
```

Comment:
Canonical imports?

Some imports isnt all that informative

Reply:
The comment can even be omitted. And I think is better to use `import bambi as bmb` in all examples (docstrings, notebooks, papers, etc).

Reply:
You're both right, I don't know what made me think that sentence was adding something haha